vertx-core/src/main/java/io/vertx/core/net/impl/NetServerImpl.java

```diff
@@ -507,13 +506,8 @@ private void bind(
     channelBalancer.addWorker(eventLoop, worker);
     ServerBootstrap bootstrap = new ServerBootstrap();
     bootstrap.group(vertx.getAcceptorEventLoopGroup(), channelBalancer.workers());
-    if (options.isSsl()) {
-      bootstrap.childOption(ChannelOption.ALLOCATOR, PartialPooledByteBufAllocator.INSTANCE);
-    } else {
-      bootstrap.childOption(ChannelOption.ALLOCATOR, VertxByteBufAllocator.POOLED_ALLOCATOR);
-    }
-
     bootstrap.childHandler(channelBalancer);
+    bootstrap.childOption(ChannelOption.ALLOCATOR, VertxByteBufAllocator.POOLED_ALLOCATOR);
```

Comment:
VertxByteBufAllocator.UNPOOLED_ALLOCATOR is no longer used and could be dropped as well.

Reply:
done, thanks